tsdb/compact.go

```diff
@@ -488,16 +488,22 @@ func (c *LeveledCompactor) Write(dest string, b BlockReader, mint, maxt int64, p
 	}
 
 	if meta.Stats.NumSamples == 0 {
-		return ulid.ULID{}, nil
```

Comment:
maybe we could leave the return statement here and just add the log message right above it, then we could remove the else statement below

reading the Prometheus codebase, I think we avoid `else` whenever it's possible to improve readability

Reply:
Since `Compact` method is using as such, i was following the format to achieve consistency. 
https://github.com/prometheus/prometheus/blob/09a31ef6cfb052b4bfc8461b40a9dfed996aaeb1/tsdb/compact.go#L432-L441

Reply:
+1 to this

Reply:
The case with `Compact` is a little different, `if else` would be required there

Reply:
done